### PR TITLE
Publish Gromacs blog post

### PR DESCRIPTION
**Pull request by clairbarrass**
Hi Lorna, Julien
This is Julien's blog post ready for checking and publishing if you are happy with it.

### CODE REVIEW

File: _posts/2020-10-27-gromacs.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+author: Julien Sindt EPCC
+categories: [news]
+---
+
+One of the upsides of working on the ARCHER2 CSE team is that,
+sometimes, one finds oneself in the interesting position of being 
+the *only* user on the ARCHER2 four-cabinet system (at least
+the only apparent user -- I'm sure that AdrianJ is lurking 
+on one of the compute nodes somewhere). For context, the ARCHER2
+four-cabinet system is roughly equivalent in CPU-count to the
```
Comment (j-sindt) [inline _posts/2020-10-27-gromacs.md:14]: "CPU-count" should be "core-count" in this sentence (but kept as CPUs in the brackets)

File: _posts/2020-10-27-gromacs.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+interesting to see how those benchmarks perform on the new system.
+Also, speaking as someone who kept their desktop on for four weeks 
+while waiting for a bubble nucleation simulation to finish running 
+(only to find that the input parameters had been wrong), I've got a 
+bit of a vested interest in how well ARCHER2 does with larger MD 
+systems.
+
+Conveniently, I've recently compiled a central version of GROMACS. So,
+a quick download of the GROMACS benchmarks, a couple of minutes of 
+writing a shell script and we're off! The smallest system (20k atoms)
```
Comment (j-sindt) [inline _posts/2020-10-27-gromacs.md:39]: "shell script" to "Slurm submission script"

File: _posts/2020-10-27-gromacs.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+<img src="{{ site.baseurl }}/img/news/2020-10-27-1400k.jpg" alt="1400k plot image" />
+
+Hmmm... 1400k is going along nicely. There's some neat linearity in its 
+performance at low node count. However, that scaling seems to disappear
+past 8 nodes (1024 cores). Performance-wise, it's looking better than 
+ARCHER on a per-core basis -- I make that a roughly 3x increase per core.
+Part of this speed-up might be down to me using the 2020 version of 
+GROMACS (whereas the HECBioSim benchmarks are run with the 2018 version).
+However, not all of that speed-up can will be due to that. The ARCHER2
```
Comment (j-sindt) [inline _posts/2020-10-27-gromacs.md:62]: "can will be due" to "will be due"

File: _posts/2020-10-27-gromacs.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+the ARCHER2 system. It passes 64 nodes... it passes 128 nodes... it
+passes 256 nodes... It reaches 512 and... crashes...
+
+...
+
+... 
+
+
+The box size is too small to be decomposed over such a large number of
+nodes...
```
Comment (j-sindt) [inline _posts/2020-10-27-gromacs.md:91]: "nodes" to "cores"

**Review by j-sindt — COMMENTED** [review]
Heya -- looks good. There's a couple of mistakes (all mine) that I spotted.

**Review by jsindt — COMMENTED** [review]
Please could you make the changes I've suggested with my other account? Happy to make those changes instead if that's easier.

**Comment by clairbarrass** [comment]
Hi Julien - your corrections now made

**Comment by clairbarrass** [comment]
Sorry, I was still processing Training registrations for the course starting this morning at 9pm last night so didn’t get round to it.
Done now.

Best wishes

Clair
--
Clair Barrass
EPCC User Support & Training Admin
+44 (0)131 651 7836

**Review by jsindt — APPROVED** [review]
Looks good to me. Thanks for the changes.

**Pull request merged**